Tests/TensorFlowTests/OperatorTests/NNTests.swift

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+import TensorFlow
+
+class NNTests: XCTestCase {
+  func testDepthToSpace() {
+    let input = Tensor<Float>([
```

Comment:
These unit tests seem a little thin.  Are there are edge cases that should be stressed?  What about `size`s other than 2?

Reply:
I think there's no edge case.

Reply:
And there's no value in testing `blockSize != 2`?  I'll take your word for it if you say so.

Reply:
I looked into `depthtospace_op_test.py` and find there's one edge case I didn't expect.
https://github.com/tensorflow/tensorflow/blob/83aecde6387dd841fe3cad77b66e5c75f82bd882/tensorflow/python/kernel_tests/depthtospace_op_test.py
I added it in https://github.com/tensorflow/swift-apis/pull/710/commits/197f59a3fafc177f0f7c79f330475acd13a77ca2.